lib/vladlev.rb

```diff
@@ -27,14 +28,7 @@ def self._normalized_distance(str1, str2, max)
     require 'ffi'
     extend ::FFI::Library
 
-    native_file_path = case
-                       when relative_exists?('levenshtein.bundle') then
-                         File.join(File.dirname(__FILE__), 'levenshtein.bundle')
-                       else
-                         File.join(File.dirname(__FILE__), 'levenshtein.so')
-                       end
-
-    ffi_lib native_file_path
+    ffi_lib C_EXT_NATIVE
```

Comment:
Returning a path in `relative_existing_path` allows us to reuse the result in here.